isochrone/src/main/java/com/graphhopper/isochrone/algorithm/Isochrone.java

```diff
@@ -187,10 +187,11 @@ private void searchInternal(int from) {
             int neighborNode = currEdge.adjNode;
             EdgeIterator iter = explorer.setBaseNode(neighborNode);
             while (iter.next()) {
-                if (!accept(iter, currEdge.edge)) {
+                if (!accept(iter)) {
                     continue;
                 }
                 // minor speed up
+                // todonow: really ?
```

Comment:
Is this a u-turn check ?

Reply:
Yes, I think so. It was basically copied from Dijkstra where we introduced this a long time ago:

https://github.com/graphhopper/graphhopper/commit/f40ef5abcbfd235717b187709217ae1458d00a4a#diff-fc38211c3f0fdf3ba26000cce27b9ce7

Reply:
Ok I removed my todo comment. Isochrone uses node-based traversal, so it is not really related to this PR (?).

Reply:
Ah no, the same check was applied for other algorithms via `AbstractRoutingAlgorithm#accept`. First I thought this is only harmful because it prevents u-turns regardless of the u-turn costs configured in  `TurnWeighting`, but for node-based traversal this check is ok and saves us calculating some edge-weights etc.

-> I have included this check in `accept` again, but only for node-based traversal.

Reply:
here: 5d616b491ebb6e21dbb0a42b8aacc2e54b12160e

Reply:
Ah, ok!